lib/systems/platforms.nix

```diff
@@ -209,6 +209,14 @@ rec {
   # Legacy attribute, for compatibility with existing configs only.
   raspberrypi2 = armv7l-hf-multiplatform;
 
+  # Nvidia Bluefield 2 (w. crypto support)
+  bluefield2 = {
+    gcc = {
```

Comment:
Does `rustc` works there?

Reply:
I compiled ripgrep for bluefield2, which first compiled the rust compiler with an aarch64 target enabled and then ripgrep. It seemed to work.